src/app/business/profile/profileCard/profile-card.component.ts

```diff
@@ -147,20 +142,22 @@ export class ProfileCardComponent implements OnInit {
         }else{
             this.isAdministrator = false;
         }
+        this.chartDatas = {
+            labels: ['Unused Capacity', 'Used Capacity'],
+            datasets: [
+                {
+                    data: [this.data.totalFreeCapacity,this.data.totalCapacity-this.data.totalFreeCapacity],
+                    backgroundColor: [
```

Comment:
please define a constants file with name=colorcode...use the name everywhere in the code...that way, we can easily support themes, skinning, changing colors, usability etc. instead of changing color codes in mulitple places in the code

Reply:
@rhsakarpos As mentioned above added to the backlog by updating #84

Reply:
LG...AB